src/components/outputScreen.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from "react";
+
+class OutputScreen extends Component {
+  state = {};
+  render() {
+    return (
+      <div class="output-screen-container">
+        <input class="output-screen" type="text" />
+      </div>
+    );
+  }
```

Comment:
- [ ] Kindly make sure you update your `class="..."` attributes to `className="..."` for React best practices. Right now the application raises an error when trying to **run the server**.